_posts/2022-04-23-creating_site_config.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: How to create web site configuration model?
+categories: [Python, Django]
+author_github: negativename
+author: Kobenko Vladislav
+---
+
+In case you need to create web site configuration model, which means, that you can only create one instance of this class.
```

Comment:
```suggestion
This article describes how to create a class with a single instance. For example, In case you want to create web site configuration model, that would hold application-wide customization like URL or logo.
```